tests/i18n/test_extraction.py

```diff
@@ -687,6 +687,25 @@ def test_all_locales(self):
         self.assertTrue(os.path.exists(self.PO_FILE_KO))
         self.assertFalse(os.path.exists('locale/_do_not_pick/LC_MESSAGES/django.po'))
 
+    def test_locales_format_valid(self):
+        """Check the standard output(stdout) message in case the format of locale is valid"""
+        out = StringIO()
```

Comment:
This should go inside the subTest. There's no reason for the output to look like this: 

AssertionError: 'invalid locale garbageValue_001' not found in 'processing locale en-IN\nprocessing locale th_TH_in\nprocessing locale aSA_tZ\nprocessing locale aZ_CyRl\ninvalid locale EN-IN\nprocessing locale EN-IN\ninvalid locale EN-in\nprocessing locale EN-in\nprocessing locale garbageValue_001\n'

Reply:
Yeah! didn't think of it. Will change it.